autoload/kite/client.vim

```diff
@@ -132,9 +132,10 @@ endfunction
 
 
 function! kite#client#request_related(json, handler)
-  let path = s:codenav_path
+  let path = s:codenav_path 
+  let timeout = 10000 "10s
   if has('channel')
-    call s:async(function('s:timer_post', [path, g:kite_long_timeout, a:json, a:handler]))
+    call s:async(function('s:timer_post', [path, timeout, a:json, a:handler]))
   else
     call kite#async#execute(s:external_http_cmd(s:base_url.path, g:kite_long_timeout, 1),
```

Comment:
Did you want to change the timeout for Neovim as well?

Reply:
Yes. How can I ensure it's also used by Neovim?

Reply:
In the `else` block, replace `g:kite_long_timeout` with `timeout`.